themes/jquery/header.php

```diff
@@ -23,7 +23,7 @@
 
 	<link rel="shortcut icon" href="<?php echo get_stylesheet_directory_uri(); ?>/i/favicon.ico">
 
-	<link rel="stylesheet" href="<?php echo get_template_directory_uri(); ?>/css/base.css?v=2">
+	<link rel="stylesheet" href="<?php echo get_template_directory_uri(); ?>/css/base.css?v=3">
```

Comment:
We have some caching issues so after changes it's the safest to bump this number, unfortunately...